docker-compose.yml

```diff
@@ -27,6 +29,7 @@ services:
       dockerfile: Dockerfile
     volumes:
       - ./src/backend:/usr/local/src/backend
+      - ./src/frontend/build:/usr/local/src/backend/static/
```

Comment:
Inspired by changes in this PR: https://github.com/azavea/echo-locator/pull/454. This way, the frontend static app should be served from django root.